platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/ConcurrentArray.java

```diff
@@ -354,7 +362,11 @@ public StandardFuture<Void> parallelTraverse(Executor executor, Consumer<T> acti
                         result.complete(null);
                     }
                 } catch (Exception e) {
-                    result.cancelWithError(e);
+                    logger.error(VIRTUAL_MERKLE_STATS.getMarker(), "Exception in parallelTraverse", e);
```

Comment:
Why `VIRTUAL_MERKLE_STATS` and not `EXCEPTION`?

Reply:
Agree, I'll change it to EXCEPTION